src/components/biggive-footer/biggive-footer.scss

```diff
@@ -82,13 +82,18 @@ nav ul li a {
   }
 }
 
+.row-logo {
```

Comment:
The goal with this and the mobile counterpart below is to visually match the previous background image approach as closely as possible. Hopefully doing it with an `<img>` means it's less brittle to work with the styles (because there aren't margins that implicitly include an image's height too etc.) and with automated a11y tooling (because it knows what text's actual background colour is).